rules/style.js

```diff
@@ -121,7 +121,11 @@ module.exports = {
     "operator-linebreak": [2, "after"],
     // Enforce padding within blocks
     // http://eslint.org/docs/rules/padded-blocks
-    "padded-blocks": [2, "never"],
+    "padded-blocks": [2, {
+      "blocks": "never",
+      "classes": "always",
```

Comment:
@Leland-Kwong @blazzy @briantrice @yanfali @paulyoung what are your thoughts on padding blocks? right now, we don't allow it. i think it may create more clarity to ensure padding.

this rule can be auto-fixed, so it's easy to make this change and update all existing code that wouldn't pass the rule.